packages/oruga/vite.config.ts

```diff
@@ -136,5 +138,29 @@ export default defineConfig(({ mode }) => ({
         coverage: {
             provider: "istanbul",
         },
+        workspace: [
+            {
+                extends: true,
+                test: {
+                    exclude: [browserTestPattern],
+                },
+            },
+            {
```

Comment:
Can you explain to me why we have two separate entries in the workspace here, with the second being an extended variation of the first?

Reply:
The two entries allow us to run most of the tests using the previous JSDOM implementation and only start up a full browser for tests that require it (`*.browser.test.{js,ts}`). The entries don't extend each other; they both [extend the root test configuration](https://vitest.dev/guide/workspace.html#configuration), which means we don't have to duplicate the `setupFiles` and `coverage` keys.

Reply:
The `setupFiles` and `coverage` seem like they _could_ be applied just to the non-browser configuration, but doing that breaks all the tests for some reason.